salt/utils/cloud.py

```diff
@@ -1541,6 +1558,15 @@ def deploy_script(host,
                     )
                 log.debug('Executed command \'{0}\''.format(deploy_command))
 
+                if force_minion_config:
+                    # TODO: this entire code block should be unnecessary if they implement
```

Comment:
My patch for salt-bootstrap #1105 has been merged.  We should pull this out before merging here.
I will prepare the update as soon as I can.